lucille-examples/lucille-distributed-example/src/main/java/com/kmwllc/lucille/distributed/QuerySolr.java

```diff
@@ -0,0 +1,39 @@
+package com.kmwllc.lucille.distributed;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import org.apache.solr.client.solrj.SolrClient;
+import org.apache.solr.client.solrj.impl.CloudHttp2SolrClient;
+import org.apache.solr.client.solrj.response.QueryResponse;
+import org.apache.solr.common.SolrDocument;
+import org.apache.solr.common.params.MapSolrParams;
+
+class QuerySolr {
```

Comment:
is QuerySolr used anywhere? I see it referenced in run.sh but only in a line that's commented out...